demo/api/src/warnings/warning-checker.console.ts

```diff
@@ -0,0 +1,79 @@
+import { BlockInput, FlatBlocks } from "@comet/blocks-api";
+import { DiscoverService } from "@comet/cms-api/lib/dependencies/discover.service";
+import { CreateRequestContext, MikroORM } from "@mikro-orm/core";
+import { InjectRepository } from "@mikro-orm/nestjs";
+import { EntityManager, EntityRepository } from "@mikro-orm/postgresql";
+import { Injectable } from "@nestjs/common";
+import { Command, Console } from "nestjs-console";
+import { v5 } from "uuid";
+
+import { Warning } from "./entities/warning.entity";
+import { WarningSeverity } from "./entities/warning-severity.enum";
+
+@Injectable()
+@Console()
+export class WarningCheckerConsole {
+    constructor(
+        private readonly orm: MikroORM,
+        private readonly discoverService: DiscoverService,
+        private readonly entityManager: EntityManager,
+        @InjectRepository(Warning) private readonly warningsRepository: EntityRepository<Warning>,
+    ) {}
+
+    @Command({
+        command: "check-warnings",
+        description: "Checks for warnings",
+    })
+    @CreateRequestContext()
+    async execute(): Promise<void> {
+        // TODO: (in the next PRs) Check if data itself is valid in the database. (Maybe some data was put into database and is not correct or a migration was done wrong)
+
+        for (const rootBlockEntity of this.discoverService.discoverRootBlocks()) {
+            const { metadata, column, block } = rootBlockEntity;
+
+            const queryBuilder = this.entityManager.createQueryBuilder(metadata.className);
+            queryBuilder.select(`id, "${column}"`).from(metadata.tableName);
```

Comment:
Could we load all root block columns of a table at once? This would reduce database calls.

Reply:
I made an own function that groups them in typescript so I can reduce database calls: 03f95243347c0f1616e700687362bccb44925aaa